test/get.sh

```diff
@@ -410,18 +410,18 @@ it_honors_the_parameter_flags_for_submodules() {
   "file://"$project_folder 1 "all" true $dest_recursive_true |  jq -e "
     .version == {ref: $(echo $project_last_commit_id | jq -R .)}
   "
-  test "$(git -C $project_folder rev-parse HEAD)" = $project_last_commit_id
-  test "$(git -C $dest_recursive_true/$submodule_name rev-list --all --count)" = 1
-  test "$(git -C $dest_recursive_true/$submodule_name/$subsubmodule_name rev-list --all --count)" = 1
+  test "$(git -C $dest_recursive_true rev-parse HEAD)" = $project_last_commit_id
```

Comment:
Was this assertion broken before? Looks like it was testing against the source repo rather than the fetched repo?

Reply:
Yep, this looked to be testing the wrong repo before.